src/themes/custom/app/shared/dso-selector/modal-wrappers/create-collection-parent-selector/create-collection-parent-selector.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+import {
+    CreateCollectionParentSelectorComponent as BaseComponent
+} from 'src/app/shared/dso-selector/modal-wrappers/create-collection-parent-selector/create-collection-parent-selector.component';
+
+@Component({
+  selector: 'ds-create-collection-parent-selector',
+  templateUrl: '../dso-selector-modal-wrapper.component.html',
+})
```

Comment:
The convention for the custom theme is to already create html and css file for the component, but add them in comments and refer back to the version in the base theme.

That means that if you enable the theme, everything looks exactly like the base theme. Then if you want to start theming the component all you need to do is swap the comments around and you can start overwriting the css or the html right away.

Also, I've noticed that using absolute paths in imports tends to cause build problems in certain scenarios that are hard to diagnose, so I'd suggest always using relative paths, even though that can lead to a ridiculous number of `../`-es, like in this case. However most IDEs can keep count for you.

```suggestion
import {
    CreateCollectionParentSelectorComponent as BaseComponent
} from '../../../../../../../app/shared/dso-selector/modal-wrappers/create-collection-parent-selector/create-collection-parent-selector.component';

@Component({
  selector: 'ds-create-collection-parent-selector',
  // styleUrls: ['create-collection-parent-selector.component.scss'],
  // templateUrl: 'create-collection-parent-selector.component.html',
  templateUrl: '../../../../../../../app/shared/dso-selector/modal-wrappers/dso-selector-modal-wrapper.component.html',
})
```

Reply:
Done, I think.